frontend/src/Settings/Notifications/Notifications/EditNotificationModalContent.js

```diff
@@ -75,7 +58,9 @@ function EditNotificationModalContent(props) {
 
         {
           !isFetching && !!error &&
-            <div>Unable to add a new notification, please try again.</div>
+            <div>
```

Comment:
Ternary.

Reply:
Gonna be honest no idea what that means / you're looking for here